### PR TITLE
Dynamically built and localized installation slides

### PR DESCRIPTION
**Pull request by jpnurmi**
The old slides were images. The new slides are built with Flutter widgets.

https://user-images.githubusercontent.com/140617/145833452-ab51e36f-df77-4dba-a556-9914e8e6dc91.mp4

Ref: #42

### CODE REVIEW

**Comment by jpnurmi** [comment]
Test coverage reduces quite a bit. The slide "framework" should have some tests.

**Comment by jpnurmi** [comment]
I've again split off the flavor translations to a separate PR #592 because now that I think of it, it would be nicer to not mix the generic non-installer-specific flavor classes with these installer-specific installation slides.

**Comment by jpnurmi** [comment]
- this depends on PR #592
- to make reviewing easier, commits marked with "[SQUASHED]" are only adding strings and assets and regenerating localizations
- the last commit is the important part of this PR

**Comment by jpnurmi** [comment]
A quick prototype with assets stolen from ubiquity-slideshow-ubuntu

Screenshot from 2022-01-19 17-35-14

**Review by oSoMoN — CHANGES_REQUESTED** [review]
This looks good to me overall, just a bunch of minor comments/suggestions.

The screenshots in the slides all look blurry to my untrained eye, but they appear to be [the original screenshots](https://git.launchpad.net/ubuntu/+source/ubiquity-slideshow-ubuntu/tree/slideshows/ubuntu/slides/screenshots) from ubiquity-slideshow-ubuntu, so that's not a regression, and it should be addressed there.

And one suggestion: could the left/right arrow keys trigger navigation through the slides?

**Comment by jpnurmi** [comment]
> And one suggestion: could the left/right arrow keys trigger navigation through the slides?

Wait a sec, now that I read this again, I misunderstood and made it wrap.

**Comment by jpnurmi** [comment]
> And one suggestion: could the left/right arrow keys trigger navigation through the slides?

Sounds good. Let's do this in the `SlideShow` widget in `ubuntu-flutter-plugins`.

**Comment by jpnurmi** [comment]
Arrow keys for SlideShow: https://github.com/canonical/ubuntu-flutter-plugins/pull/30

File: packages/ubuntu_desktop_installer/lib/slides/default_slides.dart [hunk 1]
```diff
@@ -0,0 +1,522 @@
...
+  body: (context) {
+    return SlideLayout(
+      padding: _kInsets,
+      contentAlignment: Alignment.topLeft,
+      background: FlavorImage.asset(_slideAsset('welcome')),
+      content: _SlideColumn(
+        spacing: _kLargeSpacing,
+        children: [
+          _SlideLabel.large(
+            context.lang.supportSlideDocumentation(context.flavor.name),
```
[the remark this answers is not in the document]
Comment (oSoMoN) [reply packages/ubuntu_desktop_installer/lib/slides/default_slides.dart:309]: Can we add the missing hyperlinks on the last slide (support), like in [the original ubiquity slideshow](https://git.launchpad.net/ubuntu/+source/ubiquity-slideshow-ubuntu/tree/slideshows/ubuntu/slides/gethelp.html) ?

**Comment by jpnurmi** [comment]
I knew I would one day commit the default route change. :man_facepalming:

**Comment by jpnurmi** [comment]
The snap build failure looks like a temporary hiccup:
```
Err:3 http://archive.ubuntu.com/ubuntu focal/main amd64 libfuse2 amd64 2.9.9-3
  Could not connect to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (91.189.88.142), connection timed out
```

**Review by oSoMoN — APPROVED** [review]
LGTM now, let's merge!

**Pull request merged**